core/domain/user_domain.py

```diff
@@ -806,7 +806,7 @@ def __init__(
         self.topic_ids_to_learn = topic_ids_to_learn
 
     def add_topic_id_to_learn(self, topic_id):
-        """Adds the topic id to learn list.
+        """Adds the topic id to topic IDs to learn list.
```

Comment:
Put single quotes around 'topic IDs to learn'.

Reply:
Done.